docs/toml-schema.md

```diff
@@ -430,3 +430,16 @@ allowed-merge-teams = ["awesome-team"]
 # (optional)
 merge-bots = ["homu"]
 ```
+
+### Crates.io trusted publishing
+Configure crates.io Trusted Publishing for crates published from a given repository from GitHub Actions.
+
+```toml
+[[trusted-publishing]]
```

Comment:
Nit: I'd call it `crates-io-publishing`, if in the future we need to integrate with other ecosystems.